### ISSUE TITLE
Changing a Track folders name also changes it's sorting option... 

### ISSUE THREAD

**Issue by Rowin63** [osmandapp/OsmAnd#20128]
### Description

... back to "last edited" 

### Steps to reproduce

Try it ;-) 

### Actual result

Sorting option is set back to "last modified" 

### Expected result

keep my one and only preferred option "A-Z" 

### Your Environment (required)

4.8.0#2533

**Comment by yuriiurshuliak** [comment]
The bug has been reproduced. To reproduce it, switch the "Sort by" option from "Last modified" to "A-Z" for any folder. Then, rename the folder. After renaming, the "Sort by" setting will revert from "A-Z" back to "Last modified".

OsmAnd~ 4.8.0#2533m, released: 2024-06-19

https://github.com/osmandapp/OsmAnd/assets/127092082/7c17a942-4f39-4d93-8b88-eeebede94477

**Comment by sonora** [comment]
Yes, inspecting the code, we correctly remove the sort mode for the old folder name, but fail to save the old sort mode under the new folder name. I can take a look when I get a chance.
